common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1752,6 +1752,12 @@
     },
     "sqlState" : "22003"
   },
+  "INVALID_BUCKET_COLUMN_DATA_TYPE" : {
+    "message" : [
+      "Cannot use <type> for bucket column."
```

Comment:
Could you help users, and explain why it cannot (or which types can be used) otherwise not clear how users should deal with the error.

Reply:
Changed. Is this sufficient?